docs/aced-RBAC.md

```diff
@@ -0,0 +1,379 @@
+gen3_util projects add user user@example.com ohsu-test
+```
+
+* Ony user's with the ohsu_requestor_updater_role  can approve and sign a request
+
+```text
+gen3_util access <request-ids> SIGNED
+```
+
+
+## Configuration
+
+```yaml
+      authz:
+        # policies automatically given to anyone, even if they are not authenticated
+        anonymous_policies: []
+
+        # policies automatically given to authenticated users (in addition to their other policies)
+        all_users_policies:
+        - requestor_creator  # all users can create a request for data access
+
+        groups:
+
+        # preconfigured roles for system admins
+        - name: administrators
+          policies:
+          - workspace
+          - services.sheepdog-admin
+          - data_upload
+          - requestor_updater
+          - requestor_creator
+          - requestor_reader
+          - indexd_admin
+          users: []  # PREDEFINED ADMIN USERS GO HERE
+
+
+        resources:
+        - name: workspace  # notebooks, etc
+        - name: data_file  # indexd
+        - name: sower      # jobs
+        - name: services   # sheepdog
+          subresources:
+          - name: sheepdog
+            subresources:
+            - name: submission
+              subresources:
+              - name: program
+              - name: project
+        - name: programs   # institutions
+          subresources:
+          - name: ohsu
+            subresources:  # studies
+            - name: projects
+          - name: ucl
+            subresources:
+            - name: projects
+          - name: manchester
+            subresources:
+            - name: projects
+          - name: stanford
+            subresources:
+            - name: projects
+
+
+        policies:
+
+        - id: workspace
+          description: be able to use workspace
+          resource_paths:
+          - /workspace
+          role_ids:
+          - workspace_user
+          - administrator
+          - writer
+          - reader
+
+        - id: data_upload
+          description: upload raw data files to S3
+          role_ids:
+          - writer
+          - administrator
+          resource_paths:
+          - /data_file
+
+        - id: services.sheepdog-admin
+          description: CRUD access to programs and projects
+          role_ids:
+            - administrator
+          resource_paths:
+            - /services/sheepdog/submission/program
+            - /services/sheepdog/submission/project
+
+        - id: indexd_admin
+          description: full access to indexd API
+          role_ids:
+            - administrator
+          resource_paths:
+            - /programs
+            - /data_file
+
+        # REQUESTOR POLICIES
+        - id: requestor_creator
+          description: Allows requesting access to any resource under "/programs"
+          role_ids:
+            - requestor_creator_role
+          resource_paths:
+            - /programs
+            - /sower      
```

Comment:
did you mean to include sower permissions in the 3 requestor policies?

Reply:
another nice catch.  thanks again